src/core/CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ target_link_libraries(${PROJECT_NAME}  PRIVATE phare_initializer MPI::MPI_C
 # skip mpicxx in compile command if not configured with configurator
 if (NOT phare_configurator)
      target_compile_definitions(${PROJECT_NAME} PUBLIC MPICH_SKIP_MPICXX OMPI_SKIP_MPICXX )
-endif(phare_configurator)
+endif()
```

Comment:
cmake complains as the closing tag is not the same as the opening tag, which is kinda true